web/pandas/_templates/layout.html

```diff
@@ -1,13 +1,12 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <script type="text/javascript">
-            var _gaq = _gaq || []; _gaq.push(['_setAccount', 'UA-27880019-2']); _gaq.push(['_trackPageview']);
-            (function() {
-                var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
-                ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
-                var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
-            })();
+        <script async="async" src="https://www.googletagmanager.com/gtag/js?id=G-5RE31C1RNW"></script>
+        <script>
+            window.dataLayer = window.dataLayer || [];
+            function gtag(){ dataLayer.push(arguments); }
+            gtag('js', new Date());
+            gtag('config', 'G-5RE31C1RNW');
```

Comment:
This is just using the exact same script as we have on the docs pages (generated by the theme)